Home task: HTML 02

relisov opened the pull request:
I'm almost certain such an approach was needlessly elaborate for this task, but I just wanted to fiddle with something a wee bit more advanced.

Update: fixed some of the most glaring mistakes.

Commit:
Home task: HTML 01

Commit:
Home task: HTML 02

Commit:
Home task: HTML 02

Commit:
Home task: HTML 02

Commit:
Home task: HTML 02

Commit:
Typo correction.

Commit:
Home task: HTML 02

Commit:
Revert "Home task: HTML 02"

This reverts commit 6652cf3eace2319f8e49b8af28a37011b18fad86.

Commit:
Home task: HTML 02

I'm almost certain such an approach was needlessly elaborate for this task, but I just wanted to fiddle with something a wee bit more advanced.

Commit:
Update task_02.html

Commit:
Create task_03.html

Commit:
Revert "Create task_03.html"

This reverts commit 3d1aaa947ca5560e930c806d211819cdd42acce4.

Commit:
Revert "Merge branch 'm-sc1-59-19' of https://github.com/relisov/sc.it-academy.by into m-sc1-59-19"

This reverts commit 71b85f9a3e79593fb9df8c4d283db261233b71de, reversing
changes made to f24d5dfd33e776fa34d6dcb6de7609a75cff8f58.

Commit:
Home task: HTML 03

My apologies for sending such a ludicrous amount of commits, git is quite tough to figure out for a novice like me. Anyway, here's task #3.

Commit:
Update task_03.html

Commit:
Home task: HTML 04

Another

pluhin commented:
1. However, where are files?
2. Why did you remove all existing files? How are you going work with team if you delete all project files?

ghost commented:
Good morning. I am currently trying to reorganize the mess of previous commits into something barely manageable. Unsurprisingly, these clumsy attempts failed miserably.
The files are backed up, rest assured. However, cannot for the life of me fugure out the proper command order to restore commits. In progress.

Edit: done... I suppose. Now everything _seems_ to be in order.

pluhin commented on relisov/01_HTML/task_02.html:
```diff
@@ -0,0 +1,90 @@
+
+				<p>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю cохранить его в папке python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+
+				<p class="terminal">chmod +x hello.py</p>
```
Do not use paragraph for non-human text

pluhin commented on relisov/01_HTML/task_02.html:
```diff
@@ -0,0 +1,90 @@
+
+				<p>Введите эти 4 строки:</p>
+
+				<p class="terminal">#!/usr/bin/env python<br>print 'Hello. I am a python program.'<br>name = raw_input("What is your name?")<br>print "Hello there, " + name + "!"</p>
```
Do not use paragraph for non-human text

pluhin commented on relisov/01_HTML/task_02.html:
```diff
@@ -0,0 +1,90 @@
+
+				<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+
+				<p class="terminal">greg@earth:~/python_examples$ ./hello.py<br>Hello. I am a python program.<br>What is your name? Ferd Burphel<br>Hello there, Ferd Burphel!<br>greg@earth:~/python_examples$</p>
```
Do not use paragraph for non-human text

pluhin commented on relisov/01_HTML/task_02.html:
```diff
@@ -0,0 +1,90 @@
+
+				<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и наберите следующую программу:</p>
+
+				<p class="terminal">#!/usr/bin/env python<br>for cntr in range(0,10):<br>	print cntr</p>
```
Do not use paragraph for non-human text

pluhin reviewed (changes requested):
Do not use paragraph for non-human text, could you check all files

Commit:
Home Task: HTML 01

Job's done.

Commit:
Update task_05.html

Commit:
Update task_05.html

pluhin commented on relisov/01_HTML/task_06.html:
```diff
@@ -0,0 +1,210 @@
+				<div title="Предварительно форматированный текст">
+					<h3>Предварительно форматированный текст</h3>
+
+					<span class="terminal"> x | o |<br>‐‐‐‐‐‐‐‐‐‐‐<br>   | х |<br>‐‐‐‐‐‐‐‐‐‐‐<br> o | x | o</span>
```
If you use  here ```white-space: pre;``` here, then it is not necessary to use br tag

pluhin commented on relisov/01_HTML/task_06.html:
```diff
@@ -0,0 +1,210 @@
+				</div>	
+		</div>
+	</body>
+	<script>
```
why is JS outside body

pluhin reviewed (changes requested):
All good, only one: put JS inside body

Commit:
Update: minor issues.

Turns out I actually rewrote some files while saving older versions. Lesson learned - don't mess around with git.
